code/modules/loadout/categories/pocket.dm

```diff
@@ -159,76 +98,153 @@
+/datum/loadout_item/pocket_items/dice/d6_ebony
 	name = "D6 (Ebony)"
 	item_path = /obj/item/dice/d6/ebony
 
-/datum/loadout_item/pocket_items/d6_space
+/datum/loadout_item/pocket_items/dice/d6_space
 	name = "D6 (Space)"
 	item_path = /obj/item/dice/d6/space
 
-/datum/loadout_item/pocket_items/d8
+/datum/loadout_item/pocket_items/dice/d8
 	name = "D8"
 	item_path = /obj/item/dice/d8
 
-/datum/loadout_item/pocket_items/d10
+/datum/loadout_item/pocket_items/dice/d10
 	name = "D10"
 	item_path = /obj/item/dice/d10
 
-/datum/loadout_item/pocket_items/d12
+/datum/loadout_item/pocket_items/dice/d12
 	name = "D12"
 	item_path = /obj/item/dice/d12
 
-/datum/loadout_item/pocket_items/d20
+/datum/loadout_item/pocket_items/dice/d20
 	name = "D20"
 	item_path = /obj/item/dice/d20
 
-/datum/loadout_item/pocket_items/d100
+/datum/loadout_item/pocket_items/dice/d100
 	name = "D100"
 	item_path = /obj/item/dice/d100
 
-/datum/loadout_item/pocket_items/d00
+/datum/loadout_item/pocket_items/dice/d00
 	name = "D00"
 	item_path = /obj/item/dice/d00
 
+/datum/loadout_item/pocket_items/card_binder
+	name = "Card Binder"
+	item_path = /obj/item/storage/card_binder
+
+/datum/loadout_item/pocket_items/card_deck
+	name = "Playing Card Deck"
+	item_path = /obj/item/toy/cards/deck
+
+/datum/loadout_item/pocket_items/kotahi_deck
+	name = "Kotahi Deck"
+	item_path = /obj/item/toy/cards/deck/kotahi
+
+/datum/loadout_item/pocket_items/wizoff_deck
+	name = "Wizoff Deck"
+	item_path = /obj/item/toy/cards/deck/wizoff
+
+/datum/loadout_item/pocket_items/lipstick
+	name = "Lipstick"
+	item_path = /obj/item/lipstick
+
+/datum/loadout_item/pocket_items/lipstick/get_item_information()
+	. = ..()
+	.[FA_ICON_PALETTE] = "Recolorable"
+
+/datum/loadout_item/pocket_items/lipstick/on_equip_item(
+	obj/item/lipstick/equipped_item,
+	datum/preferences/preference_source,
+	list/preference_list,
+	mob/living/carbon/human/equipper,
+	visuals_only,
+)
+	. = ..()
+	var/picked_style = style_to_style(preference_list[item_path]?[INFO_LAYER])
+	var/picked_color = preference_list[item_path]?[INFO_GREYSCALE] || /obj/item/lipstick::lipstick_color
+	if(istype(equipped_item)) // can be null for visuals_only
+		equipped_item.style = picked_style
+		equipped_item.lipstick_color = picked_color
+	equipper.update_lips(picked_style, picked_color)
+
+/// Converts style (readable) to style (internal)
+/datum/loadout_item/pocket_items/lipstick/proc/style_to_style(style)
+	switch(style)
+		if(UPPER_LIP)
+			return "lipstick_upper"
+		if(LOWER_LIP)
+			return "lipstick_lower"
+	return "lipstick"
+
+/datum/loadout_item/pocket_items/lipstick/get_ui_buttons()
+	. = ..()
+	UNTYPED_LIST_ADD(., list(
+		"label" = "Style",
+		"act_key" = "select_lipstick_style",
+		"button_icon" = FA_ICON_ARROWS_ROTATE,
+		"active_key" = INFO_LAYER,
+	))
+	UNTYPED_LIST_ADD(., list(
+		"label" = "Color",
+		"act_key" = "select_lipstick_color",
+		"button_icon" = FA_ICON_PALETTE,
+		"active_key" = INFO_GREYSCALE,
+	))
```

Comment:
Really prefer that all of these have `return .` at the end

Reply:
Per our standards we should only do it if . is used only as a variable (without ``. = ..()``)